compiler/aarch64/codegen/ARM64Debug.cpp

```diff
@@ -1489,6 +1496,32 @@ TR_Debug::print(TR::FILE *pOutFile, TR::RegisterDependencyConditions *conditions
       }
    }
 
+void
+TR_Debug::printAssocRegDirective(TR::FILE *pOutFile, TR::Instruction *instr)
+   {
+   TR_ARM64RegisterDependencyGroup * depGroup = instr->getDependencyConditions()->getPostConditions();
+
+   printPrefix(pOutFile, instr);
+   trfprintf(pOutFile, "%s", getOpCodeName(&instr->getOpCode()));
+   trfflush(pOutFile);
+
+   int first = TR::RealRegister::FirstGPR;
```

Comment:
This variable `first` is not used in this function.

Reply:
Removed `first` and `last` and rewrote the loop to use `getAddCursorForPost()` for the loop end condition.